Taskfile.yaml

```
@@ -87,6 +87,25 @@ tasks:
           go run tools/addlicense/{main,variables,msg,utils}.go -directory ./
         {{end}}
 
+  lint:
+    cmds:
+      - task: lint:doc-ru
+      - task: lint:prettier:yaml
+      - task: virtualization-controller:dvcr:_ensure:golangci-lint
+      - task: virtualization-controller:dvcr:lint
+      - task: virtualization-controller:dvcr:lint:go
+      - task: virtualization-controller:lint
+      - task: virtualization-controller:lint:go
```

Seems only 2 tasks are enough. (lint task executes lint:go task)

```suggestion
      - task: virtualization-controller:dvcr:lint
      - task: virtualization-controller:lint
```